apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -69,6 +73,24 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
+    // For AI Rubrics Pilot
+    console.log(this.level.name);
+    if (
+      experiments.isEnabled('ai-rubrics') &&
```

Comment:
note to self: add `non-ai-rubrics` to this conditional in https://github.com/code-dot-org/code-dot-org/pull/54444